CNAME

```diff
@@ -1 +0,0 @@
-maplibre.org
```

Comment:
Do you know what the function of this file was?

Reply:
Yes, that file is needed for custom domains. The Github Action will automatically generate this one though.